extensions/laravel-livewire/src/documentation/1.x.json

```
@@ -0,0 +1,48 @@
+{
+  "Start": [
+    { "url": "https://laravel-livewire.com/docs/1.x/quickstart", "title": "Quickstart1" },
```

```suggestion
    { "url": "https://laravel-livewire.com/docs/1.x/quickstart", "title": "Quickstart" },
```
Remove debug number 😄